### PR TITLE
fix for issue #20 : added loader

### PR DESCRIPTION
**Pull request by MistyRavager**
Issue: 20

#### Short description of what this resolves:
added loader which shows loading sign when data is being fetched, after that data is showed in table
**NOTE** : given API does not work, so i tried with other sample APIs i had.

#### Changes proposed in this pull request and/or Screenshots(if applicable):
- give a working API
- image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,6 +63,21 @@
             $("table tbody").append(markup);
           });
         }
+        async function getData(){
+            const res = await fetch('https://opencode-leaderboard-homewardgamer.koyeb.app/get-all-data/?page=1')
+            .then(res=>res.json())
+            .then(data =>{
+              console.log(data)
+              document.querySelector(".loading").style.display = "none" //stop the load
```
Comment (aknsal) [inline index.html:71]: You forgot to return the data here.

**Review by aknsal — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (aknsal) [reply index.html:71]: Just add `return data;` or run `addtotable()` function within the same `.then` function.

Comment (MistyRavager) [reply index.html:71]: I have added the `addToTable` function in the same ```.then``` function

**Review by aknsal — APPROVED** [review]
Great!! Thanks for contributing!

**Pull request merged**